pac4j-oidc/src/main/java/org/pac4j/oidc/profile/azuread/AzureAdProfileCreator.java

```diff
@@ -13,12 +15,19 @@
  * @author Jerome Leleu
  * @since 1.9.2
  */
-public class AzureAdProfileCreator extends OidcProfileCreator<AzureAdProfile> {
+public class AzureAdProfileCreator extends OidcProfileCreator<AzureAdProfile,AzureAdOidcConfiguration> {
```

Comment:
In fact, we don't care about the configuration type, do we really need this change?

Reply:
I can make it work without this change, but then I would store the tenant in the OidcConfiguration not in the AzureAdOidcConfiguration. I don't think that is what you want.

Can you suggest a way to make this work without this change ?

Reply:
For sure, we need the `AzureAdOidcConfiguration` to hold the tenant.

My point is: should we have the `OidcProfileCreator` parametrized with `? extends OidcConfiguration`?